testsuite/test_0245/data/test.py

```diff
@@ -0,0 +1,76 @@
+import os
+import sys
+
+arnold_plugin = os.environ['ARNOLD_PLUGIN_PATH']
+
+def find_in_file(expectedTypes, filename):
+
+    file = open(filename, 'r')
```

Comment:
it's generally recommended to use the  `with open("filename", 'r') as f:` construct so the file is closed when the code goes out of the scope, it avoids having to manually close the file.

Reply:
Thanks ! fixed